doc/source/tutorial/stats/resampling_tutorial_1.rst

```diff
@@ -0,0 +1,410 @@
+    >>> n_samples = 1000  # number of samples
+    >>> # standard normal distribution can be used, since the
+    >>> # statistic is unaffected by location and scale
+    >>> norm_dist = stats.norm()
+    >>> # Draw 1000 samples, each with 500 observations
+    >>> y = norm_dist.rvs(size=(n_observations, n_samples))
+
+    >>> # calculate the value of the statistic for each sample
+    >>> # we'll call this the "Monte Carlo null distribution"
+    >>> null_dist_mc = statistic(y)
+
+    >>> # the asymptotic null distribution is chi-squared with df=2
+    >>> null_dist = stats.chi2(df=2)
+    >>> y_grid = np.linspace(0, null_dist.isf(0.001))
+    >>> pdf = null_dist.pdf(y_grid)
+
+    >>> # compare the two
+    >>> import matplotlib.pyplot as plt
+    >>> plt.plot(y_grid, pdf)
+    >>> plt.hist(null_dist_mc, density=True, bins=100)
+    >>> plt.xlim(0, np.max(y_grid))
+    >>> plt.xlabel("Value of statistic")
+    >>> plt.ylabel("Probability Density")
+    >>> plt.legend(['Asymptotic Null Distribution', 'Monte Carlo Null Distribution (500 observations/sample)'])
+    >>> plt.show()
+
+As we can see, the *Monte Carlo null distribution*
+\ `[2] <#cite_note-2>`__ of the test statistic when samples are drawn
+according to the null hypothesis (from a normal distribution) appears to
+follow the *asymptotic null distribution* (chi-squared with two degrees
+of freedom).
+
+Note that the originally observed value of the statistic, 6.98, is
+located in the right tail of the null distribution. Random samples from
+a normal distribution usually produce statistic values less than 6.98,
+and only occasionally produce higher values. Therefore, there is rather
+low probability of observing such an extreme value of the statistic
+under the null hypothesis that the sample is drawn from a normal
+population. This probability is quantified by the *inverse survival
+function* of the null distribution:
+
+    >>> pvalue = null_dist.sf(stat1)
+    >>> message = ("Under the null hypothesis, the chance of drawing a sample "
+    ...            f"that produces a statistic value greater than \n{stat1}\n"
+    ...            f"is\n{pvalue}")
+    >>> print(message)
+    Under the null hypothesis, the chance of drawing a sample that produces a statistic value greater than
+    6.982848237344646
+    is
+    0.03045746622458189
+
+
+This is the ``pvalue`` returned by ``stats.jarque_bera``.
+
+    >>> np.testing.assert_allclose(pvalue, stats.jarque_bera(x).pvalue, rtol=1e-14)
+
+When the :math:`p`-value is small, we take this as evidence against the
+null hypothesis, since samples drawn under the null hypothesis have a
+low probability of producing such an extreme value of the statistic. For
+better or for worse, a common “confidence level” used for statistical
+tests is 0.99, meaning that the threshold for rejection of the null
+hypothesis is :math:`p \leq 0.01`. If we adopt this criterion, then the
+Jarque Bera test was inconclusive; it gives no evidence that the null
```

Comment:
Nit: everywhere else, Jarque-Bera has a hyphen

Reply:
Yup. Fortunately, I think this is the only one I missed.